docs/Changelog.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 1.4.0.9000
```

Comment:
As in the other PR, please use `1.5.0 (in progress)` to follow precedent from past releases.

Reply:
This now done.